testing/087_call_graph.c

```diff
@@ -0,0 +1,36 @@
+// objective: root and truncated nodes
+// check: 087__call__graph_8c_ab50e9ccc8158d9b5cab04dba22244fbd_cgraph.dot
+// config: DOT_GRAPH_MAX_NODES=8
+// config: GENERATE_HTML=YES
```

Comment:
Don't set this tests run in xml mode. When tests are run in html mode `doxygen tests TEST_FLAGS=--xhtml` this is added automatically.